### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by kidde60**
### Awesome books: with ES6.
**On this webpage I:**

- used JavaScript ES6.
- added modules.
- added date.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
```
Comment (elmar8287) [inline index.js]: - [ ] Please, consider removing the linter disabled. Let the linter makes its exact job - help to fix the bugs in the early stages of the project 👍

File: index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import Book from './modules/book.js';
+import Date from './modules/date.js';
+
+// UI class to handle all UI events
```
Comment (elmar8287) [inline index.js:5]: - [ ] Please, consider removing all comments to make your code clear. Actually, your code talks itself about everything happening 👍

File: README.md [hunk 3]
```diff
@@ -70,6 +96,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT.md) licensed.
```
Comment (elmar8287) [inline README.md:102]: - [ ] OPTIONAL Please, consider personalize your MIT.md file 👍

**Review by elmar8287 — APPROVED** [review]
Hi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good design 👍 
- Nice README file 👍 
- Informative PR

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- [ ] Check the OPTIONAL comments under the review.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @kidde60               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**